nix/master.nix

```diff
@@ -154,21 +154,20 @@ in
       };
 
       cachix = {
+        enable = lib.mkEnableOption "Enable Cachix integration";
```

Comment:
Maybe we should start to have some sort of changelog, because this is a minor breakage.